shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -285,3 +299,22 @@ fun String.extractShortTypeFromExpanded(): String =
      * TODO do a clean implementation using info from @context
      */
     this.substringAfterLast("/").substringAfterLast("#")
+
+fun CompactedJsonLdEntity.toKeyValues(): Map<String, Any> {
+    return this.map { (key, value) -> key to simplifyRepresentation(value) }.toMap()
```

Comment:
[nitpicking mode] could `mapValues` simplify this?

Reply:
ok